payments/utils.py

```diff
@@ -175,14 +176,20 @@ def check_payments_status(verbose=0):
             _process_payment(order, payment, verbose)
 
 
+def get_unpaid_orders(overdue=False):
+    deadline = timezone.now() - timedelta(days=UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY)
+    orders = Order.objects.filter(Q(status=Order.AWAITING) | Q(status=Order.PARTLY_PAID))
+    orders = orders.filter(Q(date_created__lt=deadline) | Q(unpaid_notification_sent_at__lt=deadline))
+    if overdue:
```

Comment:
I will do it, no problem. 
I also want to change the email template as requested in specs of the issue.